code/extensions/CommentsExtension.php

```diff
@@ -423,7 +423,18 @@ public function CommentsForm() {
 			Requirements::javascript(THIRDPARTY_DIR . '/jquery-validate/lib/jquery.form.js');
 			Requirements::javascript(COMMENTS_THIRDPARTY . '/jquery-validate/jquery.validate.min.js');
 			Requirements::javascript('comments/javascript/CommentsInterface.js');
```

Comment:
I think it's time to start combining these dependencies; There are quite a few JS files now.